runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangeConfig.java

```diff
@@ -63,4 +63,33 @@ default boolean enableOsbPrices()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+		keyName = "geBuySellData",
+		name = "",
+		description = "",
+		hidden = true
+	)
+	default String geBuySellData()
+	{
+		return "{}";
+	}
+
+	@ConfigItem(
```

Comment:
is using `ConfigItem` like this the preferred way of persisting key-value data in runelite?

Reply:
I'm not sure what the preferred way is, but this code is based off how NotesConfig.java is persisted.
Is there a better way?

Reply:
Probably a question for one of the core maintainers - @deathbeam are you able to chime in?

Reply:
For simple data yes, but this is probably misuse of the config system as it will contain A LOT of data eventually

Reply:
That was my concern, is that this is essentially using configuration to store a large data set. Should we look at writing this sort of information to the `.runelite` directory? It would mean the information is not persisted across machines but I don't see that being too much of an issue.

Reply:
We had similar discussion in regards of storing loot tracker data and the outcome from @Adam- was that it should be persisted only remotely and it would require new table in the remote RuneLite DB